src/commands/friends/FriendCommand.ts

```diff
@@ -13,3 +11,177 @@ friend list [@Joker#3650]
+        \`friend accept <mention>\` accepts a friend request from that user.\n
+        \`friend deny/decline <mention>\` denies a friend request from that user.\n
+        \`friend remove/delete <mention>\` removes that user from your friend list.\n
+        \`friend list [mention]\` lists all the user's friends, or your own if no user is given.\n
+        \`friend requests [incoming/outgoing]\` lists all of your incoming or outgoing friend requests, respectfully.
+        If no type is specified, it will list both incoming and outgoing friend requests.`,
+      examples: [
+        'friend add @Joker#3650',
+        'friend deny @Joker#3650',
+        'friend list',
+        'friend requests incoming'
+      ],
+      args: [
+        {
+          key: 'action',
+          label: 'action',
+          prompt:
+            'Would you like to `add/remove/list` friends, `accept/deny` requests, or list `requests`?',
+          type: 'string',
+          infinite: false
+        },
+        {
+          key: 'argument',
+          label: 'user or type',
+          prompt: 'Please provide a valid argument for the used action.',
+          default: '',
+          type: 'filter'
+        }
+      ]
+    })
+  }
+
+  async run(
+    msg: CommandMessage,
+    {
+      action,
+      argument
+    }: { action: string; argument: User | 'incoming' | 'outgoing' }
+  ): Promise<Message | Message[]> {
+    try {
+      switch (action.toLowerCase()) {
+        case 'add':
+          return this.sendFriendRequest(msg, argument as User)
+
+        case 'deny' || 'decline':
+          return this.denyFriendRequest(msg, argument as User)
+
+        case 'accept':
+          return this.acceptFriendRequest(msg, argument as User)
+
+        case 'remove' || 'delete':
+          return this.deleteFriend(msg, argument as User)
+
+        case 'list':
+          return this.listFriends(msg, argument as User)
+
+        case 'requests':
+          return this.listFriendRequests(msg, argument as
+            | 'incoming'
+            | 'outgoing')
+
+        default:
+          return msg.reply(`\`${action}\` is not a valid action.`)
+      }
+    } catch (err) {
+      Logger.error(err)
+      return msg.reply('Failed to create friend request.')
+    }
+  }
+
+  async sendFriendRequest(
+    msg: CommandMessage,
+    user: User
+  ): Promise<Message | Message[]> {
+    const receiver = await this.getApiUser(user.id)
+    const sender = await this.getApiUser(msg.author.id)
+
+    if (!receiver || !sender) {
+      return msg.reply(
+        'This command requires you to specify a user. Please try again.'
+      )
+    }
+
+    const { data: friendRequest } = await axios.post(
+      `${Plugin.config.api.address}/users/${receiver.id}/friends/requests?token=${Plugin.config.api.token}`,
+      {
+        user: sender,
+        receiver: receiver
+      }
+    )
+
+    if (!friendRequest) {
+      const member = msg.guild.members.get(receiver.id)
```

Comment:
Remove lines 125-130.
The user shouldnt have to be in a guild or even the same guild as the user they are trying to become friends with.